redisinsight/api/src/modules/browser/dto/keys.dto.ts

```diff
@@ -319,4 +319,13 @@ export class GetKeysWithDetailsResponse {
     description: 'Node port. In case when we are working with cluster',
   })
   port?: number;
+
+  @ApiPropertyOptional({
+    type: Number,
+    description:
+      'The maximum number of results.'
+      + ' For RediSearch this number is a value from "FT.CONFIG GET maxsearchresults" command.'
+      + ' If FT.CONFIG unavailable the number is 10.000 by default for RediSearch',
```

Comment:
`If FT.CONFIG unavailable the number is 10.000 by default for RediSearch`
Is this this true?

Reply:
This is for Cloud databases. Other databases may have differences.
Removed.